coderd/database/queries/frobulators.sql

```diff
@@ -0,0 +1,13 @@
+-- name: GetFrobulators :many
+SELECT *
+FROM frobulators
+WHERE user_id = $1 AND org_id = $2;
+
+-- name: InsertFrobulator :one
+INSERT INTO frobulators (id, user_id, org_id, model_number)
+VALUES (gen_random_uuid(), $1, $2, $3)
```

Comment:
We always generate the uuid in golang. It really helps in some cases where you want to specify the uuid in a test, and not have to `insert`, then `fetch` to get the uuid of the object you just made.
```suggestion
VALUES ($1, $2, $3, $4)
```